webview/__init__.py

```diff
@@ -96,24 +91,39 @@ def _create_children(other_windows):
     guilib = initialize(gui)
 
     for window in windows:
-        window._initialize(guilib, multiprocessing, http_server)
+        window._initialize(guilib, _multiprocessing, http_server)
 
     if len(windows) > 1:
-        t = Thread(target=_create_children, args=(windows[1:],))
+        t = threading.Thread(target=_create_children, args=(windows[1:],))
         t.start()
 
     if func:
         if args is not None:
             if not hasattr(args, '__iter__'):
                 args = (args,)
-            t = Thread(target=func, args=args)
+            t = threading.Thread(target=func, args=args)
         else:
-            t = Thread(target=func)
+            t = threading.Thread(target=func)
         t.start()
 
     guilib.create_window(windows[0])
 
 
+def start(*args,**kwargs):
+    block = kwargs.get('block', True)
+    daemon = kwargs.pop('daemon', True)
```

Comment:
Let's have only one argument `daemon` that would be the new name for block. Ie. the default would be `daemon=False` (start is blocking) and with True it would be not block until joined.

Reply:
https://github.com/r0x0r/pywebview/pull/632#discussion_r515735465